src/pages/search/Search.tsx

```diff
@@ -55,10 +59,8 @@ const Search: React.FC = () => {
   }
   const search = (event: any) => {
     if (event.target.value.length > 0) {
-      setIsSearching(true);
```

Comment:
This seems to be a bad idea -- it now shows all provisions when landing on the page without entering a search option. Instead, keep this (and line 61 below), and simply render the segment chooser outside of the if block completely.